TerminalReuse.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+CD_CMD="cd "\\\"$(pwd)\\\"""
+if echo "$SHELL" | grep -E "/fish$" &> /dev/null; then
+	CD_CMD="cd "\\\"$(pwd)\\\"""
```

Comment:
In bash, you can check for partial matches using a "a POSIX extended regular expression pattern" via `if [[ "$SHELL" =~ fish ]];`. See also https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Conditional-Constructs.

That said, the branched version does the same as the non-branched?

Reply:
> That said, the branched version does the same as the non-branched?

Oh, yeah, duh... forgot to clean that up.

I also don't understand the reason for the original check for fish. I use fish all the time and you can just chain using `&&` just like bash/zsh/sh. Although I do remember `; and` being a thing ~ 8 years ago. (edit: looked it up, it was added in fish 3).